README.rst

```diff
@@ -122,6 +122,18 @@ Install spatialite and sqlite:
                             gdal-bin libproj-dev libgeos-dev libspatialite-dev \
                             fping
 
+Quickstart
+----------
+
+1. Setting openwisp-config management_interface option
+
+* Make sure the management_interface setting of openwisp-config is set to the interface which is reachable by openwisp.
```

Comment:
flag management_interface as variable please, remember restructured text needs double slant quotes:

```
``management_interface``
```

Applies to all the other occurrences.

Reply:
Surely would update every requested change soon. Thanks a lot, would follow these rules for the other points.